app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
+
+    if len(command_list) < 3:
+        return
+
+    cmd = command_list[0]
+    file_name_in = command_list[1]
+    file_name_out = command_list[2]
+
+    if file_name_in == file_name_out or cmd != "cp":
+        return
+
+    try:
+        open(file_name_in)
```

Comment:
I think we shouldn't use try except for memoization because try except use too much memory. But maybe a mentor will help us with this later

Reply:
Most likely, it depends on the task to be performed and who will use your application